plugin/v2/plugin_linux.go

```diff
@@ -136,5 +139,32 @@ func (p *Plugin) InitSpec(execRoot string) (*specs.Spec, error) {
 		p.modifyRuntimeSpec(&s)
 	}
 
+	if userns.RunningInUserNS() {
+		for i := range s.Mounts {
+			m := &s.Mounts[i]
+			var isBind bool
+			for _, o := range m.Options {
+				switch o {
+				case "bind", "rbind":
+					isBind = true
```

Comment:
Looks like we can break the loop here? (perhaps even continue the outer loop)

Reply:
fixed